content/posts/react-query-and-type-script/index.mdx

```diff
@@ -344,6 +344,23 @@ function useGroup(id: number | undefined) {
 }
 ```
 
+<Aside title="Update" icon="bell">
+
+Since v5.25, You can use [`skipToken`](https://tanstack.com/query/latest/docs/framework/react/guides/disabling-queries#typesafe-disabling-of-queries-using-skiptoken) to disable a query while having good type safety in `queryFn`:
```

Comment:
```suggestion
Since v5.25, You can use a [`skipToken`](https://tanstack.com/query/v5/docs/framework/react/guides/disabling-queries#typesafe-disabling-of-queries-using-skiptoken) to disable a query while having good type safety in `queryFn`:
```